Tasks/VsTest/vstest.ts

```diff
@@ -300,13 +300,17 @@ function getVSTestLocation(vsVersion: number): string {
     if (vstestLocationMethod.toLowerCase() === 'version') {
         let vsCommon: string = tl.getVariable('VS' + vsVersion + '0COMNTools');
         if (!vsCommon) {
-            throw(new Error(tl.loc('VstestNotFound', vsVersion)));
+            throw (new Error(tl.loc('VstestNotFound', vsVersion)));
         } else {
             return path.join(vsCommon, '..\\IDE\\CommonExtensions\\Microsoft\\TestWindow\\vstest.console.exe');
         }
-    } else if(vstestLocationMethod.toLowerCase() === 'location') {
+    } else if (vstestLocationMethod.toLowerCase() === 'location') {
         if (!pathExistsAsFile(vstestLocation)) {
-            throw(new Error(tl.loc('AccessDeniedToPath', vstestLocation)));
+            if (pathExistsAsDirectory(vstestLocation)) {
+                return path.join(vstestLocation, 'vstest.console.exe');
+            } else {
+                throw (new Error(tl.loc('AccessDeniedToPath', vstestLocation)));
```

Comment:
>throw (new Error(tl.loc('AccessDeniedToPath' [](start = 16, length = 44)

this prints %s path does not exist. "AccessDeniedToPath" name is confusing. As this error can arise due to 2 reasons: 1. such path not there and 2. Access to the path is denied. Can we have a common string name and message for both.